release-notes.txt

```diff
@@ -1,3 +1,30 @@
+OpenMDAO Version 1.7.0 Alpha Release Notes
+June 30, 2016
+
+Backwards-Incompatible Change:
+* Change fd_options to be more self consistent, changed name to deriv_options.
```

Comment:
1. 'fd_options' is now 'deriv_options' in systems.
2. New 'type' option allows the user to select between analytic, finite difference, or complex step.
3. The 'force_fd' option is no longer needed because of type.
4. The 'complex_step' choice has been removed from the 'form' option.
5. The 'step_type' option is now 'step_calc'.
6. New explicit options 'check_type', 'check_form', 'check_step_calc', and 'check_step_size' allow you to control the fd check during check_partial_derivatives.
7. Old code will still work, but will raise deprecation warnings.